### PR TITLE
fix: Update interpreters option for riot list to sort versions and output format

### PR DESCRIPTION
**Pull request by Yun-Kim**
The `riot list -i` flag output was updated to have a newline-separated format instead of comma-separated, and to be sorted by version for consistency.

### CODE REVIEW

File: tests/data/diff_pys_riotfile.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+from riot import Venv
+
+venv = Venv(
+    venvs=[
+        Venv(
+            name="test",
+            command="exit 0",
+            pys=["2.7", "3.5", "3.6", "3.7", "3.8", "3.9"],
```
Comment (brettlangdon) [inline tests/data/diff_pys_riotfile.py:8]: can we make this unsorted to verify that it does get sorted?

Comment (Yun-Kim) [reply tests/data/diff_pys_riotfile.py]: Thanks for the good catch!

File: tests/data/diff_pys_riotfile.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from riot import Venv
+
+venv = Venv(
+    venvs=[
+        Venv(
+            name="test",
+            command="exit 0",
+            pys=["3.5", "3.8", "3.6", "3.9", "3.7", "2.7"],
```
Comment (brettlangdon) [inline tests/data/diff_pys_riotfile.py:8]:
```suggestion
            # DEV: purposely out of order so we can verify sorting in tests
            pys=["3.5", "3.8", "3.6", "3.9", "3.7", "2.7"],
```



?

**Review by brettlangdon — APPROVED** [review]

File: riot/riot.py [hunk 3]
```diff
@@ -7,6 +7,8 @@
 import itertools
 import logging
 import os
+from packaging.version import parse
```
Comment (brettlangdon) [inline riot/riot.py:10]: is this used?

**Review by brettlangdon — APPROVED** [review]

**Pull request merged**